internal/node/node_signals_linux.go

```diff
@@ -0,0 +1,17 @@
+//go:build linux
+
+package nexnode
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func (n *Node) installSignalHandlers() {
+	n.log.Debug("installing signal handlers")
+	// both firecracker and the embedded NATS server(s) register signal handlers... wipe those so ours are the ones being used
+	signal.Reset(syscall.SIGINT, syscall.SIGTERM, syscall.SIGUSR1, syscall.SIGUSR2, syscall.SIGHUP)
```

Comment:
the only difference between linux and windows here is that linux wipes `SIGUSR1` and `SIGUSR2` .. do we need this or can this code be shared?

Reply:
No dice :/

No `SIGUSR1` or `SIGUSR2` on Windows.